index.js

```diff
@@ -1,24 +1,26 @@
 // base* encoding
-// Credits to https://github.com/cryptocoinjs/bs58
+// Forked from https://github.com/cryptocoinjs/bs58
+// Originally written by Mike Hearn for BitcoinJ
+// Copyright (c) 2011 Google Inc
+// Ported to JavaScript by Stefan Thomas
+// Merged Buffer refactorings from base58-native by Stephen Pair
+// Copyright (c) 2013 BitPay Inc
 
 module.exports = function base (ALPHABET) {
   var ALPHABET_MAP = {}
   var BASE = ALPHABET.length
   var LEADER = ALPHABET.charAt(0)
 
-  // pre-compute lookup table
-  for (var i = 0; i < ALPHABET.length; i++) {
+  for (var i = 0; i < ALPHABET.length; ++i) {
     ALPHABET_MAP[ALPHABET.charAt(i)] = i
   }
 
   function encode (buffer) {
     if (buffer.length === 0) return ''
 
-    var i, j
-    var digits = [0]
-
-    for (i = 0; i < buffer.length; i++) {
-      for (j = 0; j < digits.length; j++) digits[j] <<= 8
+    var i, j, digits = [0]
```

Comment:
Looks like the issue was fixed in eslint. Going to publish a new version of standard now.